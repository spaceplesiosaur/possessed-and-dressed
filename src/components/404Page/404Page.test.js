import React from 'react';
import ErrorPage from './404Page';
import { shallow } from 'enzyme';

it('should render with correct data', () => {
  const wrapper = shallow(<ErrorPage />)

  expect(wrapper).toMatchSnapshot()
})
