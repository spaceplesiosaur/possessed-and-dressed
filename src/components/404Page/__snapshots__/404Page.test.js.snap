// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render with correct data 1`] = `
<p
  className="error-page"
>
  404 - This page is not found!
</p>
`;
